src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -88,6 +88,8 @@ public void onTraversalStart() {
         GATKVCFConstants.MUTECT_FILTER_NAMES.stream().map(GATKVCFHeaderLines::getFilterLine).forEach(headerLines::add);
 
         headerLines.addAll(getDefaultToolVCFHeaderLines());
+        headerLines.add(new VCFFilterHeaderLine(Mutect2FilteringEngine.DUPLICATED_EVIDENCE_FILTER_NAME, "evidence for alt allele is overrepresented by apparent duplicates"));
```

Comment:
@takutosato In light of PR #3172, you should move `DUPLICATED_EVIDENCE_FILTER_NAME` to the M2 filters sections in `GATKVCFConstants` and add it to the `MUTECT_FILTER_NAMES` list.  Then, in the `M2-related filters` section of the `static` block in `GATKVCFHeaderLine` you should `addFilterLine` for this new filter.